tests/test_tablib.py

```diff
@@ -1117,6 +1118,68 @@ def test_xlsx_cell_values(self):
             data = tablib.Dataset().load(fh)
         self.assertEqual(data.headers[0], 'Hello World')
 
+    def test_xlsx_export_set_escape_formulae(self):
+        """
+        Test that formulae are sanitised on export.
+        """
+        data.append(('=SUM(1+1)',))
+        _xlsx = data.export('xlsx')
+
+        # read back using openpyxl because tablib reads formulae as values
+        wb = load_workbook(filename=BytesIO(_xlsx))
+        ws = wb.active
+        self.assertEqual('=SUM(1+1)', ws['A1'].value)
+
+        _xlsx = data.export('xlsx', escape=True)
+        wb = load_workbook(filename=BytesIO(_xlsx))
+        ws = wb.active
+        self.assertEqual('SUM(1+1)', ws['A1'].value)
```

Comment:
Let's use `wb` directly and avoid a temp variable only used once:

```suggestion
        self.assertEqual('SUM(1+1)', wb.active['A1'].value)
```

Reply:
Done